src/inmanta/protocol/openapi/converter.py

```diff
@@ -124,7 +124,7 @@ def get_swagger_html(self, openapi_spec: str) -> str:
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <script src="//unpkg.com/swagger-ui-dist@^3.25.0/swagger-ui-bundle.js"></script>
-        <link rel="stylesheet" href="//unpkg.com/swagger-ui-dist@^3.25.0/swagger-ui.css" />
+        <link rel="stylesheet" href="//unpkg.com/swagger-ui-dist@^4.11.1/swagger-ui.css" />
```

Comment:
The `script` and the `stylesheet` version should match.
I think there is also a `latest` tag that you can use, but a specific version is also fine.

If you check the page from the `/docs` endpoint and it looks ok, you don't have to do any special testing.

Reply:
> If you check the page from the /docs endpoint and it looks ok, you don't have to do any special testing.

I can only do this after it is merged in and the nightly build for documentation has run, right ?

Reply:
Not exactly. This is served by an endpoint on the server, so you only need a server (even one started during the tests is fine) with your patch applied to it.
I checked it, and it's fine (looks almost exactly like the previous version)